lib/modules/manager/ocb/readme.md

```diff
@@ -0,0 +1,21 @@
+This manager allows updating dependencies defined in build definitions for the [OpenTelemetry Collector Builder (ocb)](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder).
+
+The `ocb` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+
+```json title="If your builder files are named like foo-builder.yml or builder.yaml"
+{
+  "ocb": {
+    "fileMatch": ["builder.ya?ml$"]
+  }
+}
+```
+
+Supported dependencies and their respective `depType`s are:
+
+| Name           | depType      |
+| -------------- | ------------ |
+| Base collector | `collector`  |
```

Comment:
```suggestion
| base collector | `collector`  |
```

Is this supposed to be capitalized? It looks a bit funny because the other items in this column are all lowercase. But maybe this term is a proper noun, and needs to be capitalized...

Reply:
🤷‍♂️